specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/examples/SecurityPolicies_Create.json

```diff
@@ -92,15 +92,15 @@
             ]
           },
           "deploymentStatus": "NotStarted",
-          "provisioningState": "Creating"
+          "provisioningState": "Created"
```

Comment:
I checked the different values for provisioningState and only the below appear. Is "Created" actually something we return?

        Succeeded 
        Failed 
        Updating 
        Deleting
        Creating

Reply:
changed all created example to Succeeded